modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -696,6 +696,47 @@ paths:
                     description:
                         default response
             tags: ["AssetLibrary"]
+    "/asset-libraries/by-external-reference-code/{externalReferenceCode}/pinned-by-me":
+        delete:
+            description:
+                Unpins the asset library.
+            operationId: deleteAssetLibraryByExternalReferenceCodePinnedByMe
+            parameters:
+                -   in: path
+                    name: externalReferenceCode
+                    required: true
+                    schema:
+                        type: string
+            responses:
+                204:
+                    content:
+                        application/json: {}
+                        application/xml: {}
+                    description:
+                        default response
+            tags: ["AssetLibrary"]
+        put:
+            description:
+                Pins the asset library.
+            operationId: putAssetLibraryByExternalReferenceCodePinnedByMe
+            parameters:
+                -   in: path
+                    name: externalReferenceCode
+                    required: true
+                    schema:
+                        type: string
+            responses:
+                200:
+                    content:
+                        application/json:
+                            schema:
+                                $ref: "#/components/schemas/AssetLibrary"
+                        application/xml:
+                            schema:
+                                $ref: "#/components/schemas/AssetLibrary"
+                    description:
+                        default response
+            tags: ["AssetLibrary"]
```

Comment:
There was a misundertanding here. The "pins" name here was fine; I suggested "pinned-by-me" only for the other resource (`/asset-libraries/pinned-by-me`).